### ISSUE TITLE
Feature Request: Partially expand item

### ISSUE THREAD

**Issue by lholden** [foundryvtt/dnd5e#4411]
Right now when you expand an item in your inventory it displays both the Activities, the description, and all of the traits on that item. This can be very handy when first leveling up or setting up a class. But it can also take up a lot of screen space when you just want to keep the items expanded to access their Activities. This is especially true for items that have extremely long descriptions 

![image](https://github.com/user-attachments/assets/6d4aaa0d-887f-4c02-b145-ce5a087e59b6)

It would be nice if one could just display the activities. Perhaps make the expand button a 3 way toggle such that you get 1. Nothing, 2. Just Activities, 3. Everything.

![activities](https://github.com/user-attachments/assets/27211ecd-5d31-4c9f-9cdd-1a8a445b8a6d)

**Comment by etiquettestartshere** [comment]
Personally, I always want to see the description, *especially* on spells. I love how it works currently, and would be a little sad to find that getting to the item description is behind an extra click. If this is implemented, perhaps a second button or a shift-click to open just the activities could be a compromise.

**Comment by M0nk3yy** [comment]
Yeah I don't know what the solution is here but I definitely feel this. I was hoping to be able to have Monks Focus permanently expanded so I could always easily access the different features, but that has a LOT of text and nowhere to hide it. In the meantime it will be neater for me to just have separate features.
